examples/trials/network-morphism/FashionMNIST-pytorch.py

```diff
@@ -0,0 +1,244 @@
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import argparse
+import logging
+import os
+import pickle
+import sys
+import time
+
+import numpy as np
+import torch
+import torch.backends.cudnn as cudnn
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.onnx
+import torch.optim as optim
+import torchvision
+import torchvision.transforms as transforms
+from torch.autograd import Variable
+
+import nni
+import utils
+from utils import EarlyStopping
+
+# set the logger format
+log_format = "%(asctime)s %(message)s"
+logging.basicConfig(
+    filename="networkmorphism.log",
+    filemode="a",
+    level=logging.INFO,
+    format=log_format,
+    datefmt="%m/%d %I:%M:%S %p",
+)
+# set the logger format
+logger = logging.getLogger("FashionMNIST-network-morphism")
+
+
+def get_args():
+    parser = argparse.ArgumentParser("FashionMNIST")
+    parser.add_argument("--batch_size", type=int, default=128, help="batch size")
+    parser.add_argument("--optimizer", type=str, default="SGD", help="optimizer")
+    parser.add_argument("--epoches", type=int, default=200, help="epoch limit")
+    parser.add_argument("--learning_rate", type=float, default=0.001, help="learning rate")
+    parser.add_argument("--time_limit", type=int, default=0, help="time limit")
+    parser.add_argument("--cutout", action="store_true", default=False, help="use cutout")
+    parser.add_argument("--cutout_length", type=int, default=8, help="cutout length")
+    parser.add_argument(
+        "--model_path", type=str, default="./", help="Path to save the destination model"
+    )
+    args = parser.parse_args()
+    return args
+
+
+trainloader = None
+testloader = None
+net = None
+criterion = None
+optimizer = None
+device = "cuda" if torch.cuda.is_available() else "cpu"
+best_acc = 0.0
+args = get_args()
+
+
+def build_graph_from_json(ir_model_path):
+    """ build model from json represtation 
+    """
+    graph = pickle.load(open(ir_model_path, "rb"))
+    logging.debug(graph.operation_history)
```

Comment:
I am a little confuse about that,  the function name is "build graph from json", but it seems you build graph from pickle file path???